.chloggen/model-nfs.yaml

```
@@ -0,0 +1,22 @@
+# Use this changelog template to create an entry for release notes.
+#
+# If your change doesn't affect end users you should instead start
+# your pull request title with [chore] or use the "Skip Changelog" label.
+
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: enhancement
+
+# The name of the area of concern in the attributes-registry, (e.g. http, cloud, db)
+component: nfs
+
+# A brief description of the change. Surround your text with quotes ("") if it needs to start with a backtick (`).
+note: Add rpc.onc.* / rpc.nfs.* model to support new metric.nfs.*/metric.nfsd.*
```

Suggested change to the changelog note to mention the metrics first and some formatting changes since we know it'll be rendered into Markdown.

```suggestion
note: Add `nfs.*`/`nfsd.*` metrics, along with supporting `rpc.onc.*`/`rpc.nfs.*` attributes.
```